Project.toml

```diff
@@ -6,6 +6,7 @@ version = "0.5.2"
 [deps]
 CUDA = "052768ef-5323-5732-b1bb-66c8b64840ba"
 MPI = "da04e1cc-30fd-572f-bb4f-1f8673147195"
+Requires = "ae029012-a4dd-5104-9daa-d747884805df"
```

Comment:
Maybe set it up so it can use package extensions on Julia 1.9:
https://pkgdocs.julialang.org/v1/creating-packages/#Conditional-loading-of-code-in-packages-(Extensions)

You can see how MPI.jl does it with CUDA.jl here: https://github.com/JuliaParallel/MPI.jl/blob/master/ext/CUDAExt.jl